core/modules/translation/translation.admin.inc

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Administration form for node settings for content translation.
+ */
+
+ /**
+  * Builds the configuration form for node language configuration.
+  */
+function translation_node_configure_form() {
+
+  $form = array(
+    '#submit' => array('translation_node_configure_form_submit'),
+  );
+
+  $form['translation_hide_translation_links'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Hide content translation links'),
```

Comment:
Let's flip this checkbox around so it's "Show content translation links" and is checked by default. That way we don't have the awkward "enable to disable" concept here.

Reply:
I've added an upgrade path and flipped the messaging and logic to `Show content translation links`.